libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFPushNotificationManager.h

```diff
@@ -45,6 +45,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nullable, nonatomic, strong) NSDictionary* customPushRegistrationBody;
 
+/** Use this flag to indicate if the notifications should be encrypted. NO by default.
```

Comment:
Add a comment here that this flag should always be set to `NO` for `Apex push notifications`.